tests/unittests/test_command_line_processing.py

```diff
@@ -59,11 +60,24 @@ def test_non_existent_session_file_returns_EX_NOINPUT(local_unittest_config):
     assert cosmic_ray.cli.main(["exec", str(local_unittest_config), "foo.session"]) == ExitCode.NO_INPUT
 
 
-def test_non_existent_config_file_returns_EX_NOINPUT(session, local_unittest_config):
-    cosmic_ray.cli.main(["init", local_unittest_config, str(session)])
+def test_non_existent_config_file_returns_EX_NOINPUT(session, local_unittest_config,force):
+    cosmic_ray.cli.main(["init", local_unittest_config, str(session),force])
     assert cosmic_ray.cli.main(["exec", "no-such-file", str(session)]) == ExitCode.CONFIG
 
 
+
+def test_init_with_existing_results_no_force(session, local_unittest_config,force):
+    """Test that init exits without reinitializing when results exist and force=False"""
+    with use_db(session) as database:
+        database.num_results = 1  # Simulate existing results
```

Comment:
You can't just set `num_results` like this. The `WorkDB` class won't allow it. 

You clearly haven't run these tests because none of them run. Please don't resubmit until the tests actually run.

Reply:
sorry i commited the wrong branch will fix the changes